[PATCH] fix mbtheme build issue when png support is disabled

DEFAULTTHEME{,NAME} is not defined when png support is disabled, leading
to compile errors in mbtheme.c:

mbtheme.c: In function 'show_parse_error':
mbtheme.c:2422:28: error: 'DEFAULTTHEME' undeclared (first use in this function)
mbtheme.c:2422:28: note: each undeclared identifier is reported only once for each function it appears in
mbtheme.c: In function 'mbtheme_init':
mbtheme.c:2448:30: error: 'DEFAULTTHEME' undeclared (first use in this function)
mbtheme.c:2470:25: error: 'DEFAULTTHEMENAME' undeclared (first use in this function)

Fix it by unconditionally defining them as the values used are the same
in png / !png cases anyway.

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 src/structs.h |    9 ---------
 1 file changed, 9 deletions(-)

Index: matchbox-window-manager-1.2/src/structs.h
===================================================================
--- matchbox-window-manager-1.2.orig/src/structs.h
+++ matchbox-window-manager-1.2/src/structs.h
@@ -81,8 +81,6 @@
 
 #define GENERIC_ICON        PKGDATADIR "/mbnoapp.xpm"
 
-#ifdef MB_HAVE_PNG
-
 #define DEFAULT_THEME       DATADIR "/themes/Default/matchbox/theme.xml"
 
 #define DEFAULTTHEME       DATADIR "/themes/Default/matchbox/theme.xml"
@@ -91,13 +89,6 @@
 
 #define DEFAULTTHEMENAME  "Default"
 
-#else
-
-#define DEFAULT_THEME       DATADIR "/themes/Default/matchbox/theme.xml"
-#define DEFAULT_THEME_NAME  "Default"
-
-#endif
-
 #define CONFDEFAULTS       PKGDATADIR "/defaults"
 
 /* Simple Macros  */
